BCP195bis/draft-ietf-uta-rfc7525bis.md

```diff
@@ -696,7 +698,7 @@ Rationale: the extension does not apply to the AEAD
 The recommendations of this document primarily apply to the implementation and deployment of application protocols that are most commonly used with TLS and DTLS on the Internet today.  Examples include, but are not limited to:
 
 
-* Web software and services that wish to protect HTTP traffic with TLS.
+* Web software and services that wish to protect HTTP traffic with TLS. (HTTP implementations not intended for use in the World Wide Web, e.g. minimal implemenations deployed only on constrained devices, are not necessarily included in this applicability statement.)
```

Comment:
This is a significant exception, I don't think it belongs in parenthesis and maybe it should get its own bullet.